client/packages.go

```diff
@@ -103,6 +101,12 @@ type SnapHealth struct {
 	Code      string        `json:"code,omitempty"`
 }
 
+type SnapRefreshInhibit struct {
+	// ProceedTime is the time after which a pending refresh is forced for a
+	// running snap in the next auto-refresh.
```

Comment:
maybe needs an update to reflect that ProceedTime may be in the past, if a refresh is still pending and the snap applications are being monitored.

Reply:
updated, thank you!